apps/src/aiTutor/views/legacyLabs/AiTutorSidebar.module.scss

```diff
@@ -38,5 +32,29 @@
   background-color: white;
   cursor: pointer;
   border-radius: 0;
-  border: 1px solid lightgrey;
+  border-bottom: 2px solid lightgrey;
+
+  .icon-code {
+    color: var(--text-success-primary-fixed);
+  }
+
+  .icon-lightbulb {
+    color: var(--text-warning-primary-fixed);
+  }
+
+  .icon-book {
+    color: var(--text-info-primary-fixed);
+  }
+
+  .icon-brain {
+    color: purple;
+  }
+
+  .icon-bug {
+    color: red;
+  }
+
+  .icon-star {
+    color: goldenrod;
+  }
```

Comment:
@kelbyhawn looping you in since I think you've been working on UI polish for the AI Tutor chat window. With broad Tutor we realized that not all the default suggested prompts we show in levels make sense for standalone projects. We've added in these placeholder other options for suggested prompts in those cases. This means there's some duplicate styles floating around for the suggested prompts in the sidebar and the chat window, which we should keep in sync or consolidate. Our designs are still in flight a bit so I don't think we need to over-standardize yet just wanted to put it on your radar!

Reply:
Cool, thanks for letting me know!